target/linux/ramips/dts/mt7628an_tplink_tl-wr902ac-v4.dts

```diff
@@ -0,0 +1,112 @@
+#include "mt7628an_tplink_8m.dtsi"
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "tplink,tl-wr902ac-v4", "mediatek,mt7628an-soc";
+	model = "TP-Link TL-WR902AC v4";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		button-reset {
+			label = "reset";
+			gpios = <&gpio 38 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		button-sw1 {
+			label = "sw1";
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_0>;
+		};
+
+		button-sw2 {
+			label = "sw2";
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_1>;
+		};
+
+		button-wps {
+			label = "wps";
+			gpios = <&gpio 41 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_lan {
+			label ="green:lan";
+            color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_LAN;
+			gpios = <&gpio 5 GPIO_ACTIVE_HIGH>;
+		};
+
+		led_power: power {
```

Comment:
it'd be consistent if you called the led itself `led_power` rather than `power` (not talking about the label currently called `led_power`).

Maybe it's easier to just have `led-1`, `led-2`, `led-3`, etc for node names since nobody sees them, and keep `led_power` as a label.